react-frontend/src/components/Learning/contactUs.js

```diff
@@ -0,0 +1,73 @@
+import React, { useContext } from 'react';
+import { Col, Row } from 'react-flexbox-grid';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEnvelope } from '@fortawesome/free-regular-svg-icons';
+import Query from '../Query';
+import LEARNING_ADMINS_QUERY from '../../queries/learning/learningAdmins';
+import { AppConfigContext } from '../../providers/AppConfig';
+
+import { Heading } from '../StyleComponents/headings';
+import {
+  BannerHorizontalContainer,
+  LearningSideImageTextBox,
+} from '../StyleComponents/pageContent';
+
+function ContactUs() {
+  const strapiMediaUrl = useContext(AppConfigContext)['state'][
+    'strapiMediaUrl'
+  ];
+  return (
+    <BannerHorizontalContainer>
+      <Query query={LEARNING_ADMINS_QUERY}>
+        {({ data: { learningAdmins } }) => {
+          return (
+            <Row middle="xs">
+              <Col xs={12} lg={6}>
+                <LearningSideImageTextBox>
+                  <p>
+                    Is there a course you wish we offered? Got an idea for an
+                    event?
+                  </p>
+                  <Heading>We'd love to hear from you.</Heading>
+                </LearningSideImageTextBox>
+              </Col>
+              {learningAdmins.map((admin, i) => {
+                return (
+                  <Col xs={12} sm={6} lg={3} key={i}>
+                    <div style={{ textAlign: 'left', marginTop: '16px' }}>
+                      <img
+                        src={strapiMediaUrl + admin?.HeadShot?.url}
+                        style={{
+                          width: '100%',
+                          borderRadius: '18px',
+                          maxWidth: '300px',
+                        }}
+                      />
+                      <p style={{ marginBottom: '0', maxWidth: '300px' }}>
```

Comment:
Sorry, just one more little thing to match spacing in wireframe -- could you add `marginTop: '16px'` to this paragraph?

![space](https://user-images.githubusercontent.com/16946297/124650219-56341000-de4e-11eb-829e-c9aa35e2e3ad.png)

Reply:
added 16 px marging to the bottom